python/tvm/contrib/rocm.py

```diff
@@ -0,0 +1,28 @@
+# tvm.contrib.rocm
+"""Utility to convert object file to HSA Code Object
```

Comment:
You can simply change the comment about rocm related utilities, in case we put more things in this file